docs/sources/http_api/data_source.md

```diff
@@ -497,3 +497,110 @@ Content-Type: application/json
 `GET /api/datasources/proxy/:datasourceId/*`
 
 Proxies all calls to the actual data source.
+
+## Query a data source by id
+
+Queries a data source having backend implementation.
+
+`POST /api/tsdb/query`
+
+> **Note:** Most of Grafana's builtin data sources have backend implementation.
+
+**Example Request**:
```

Comment:
```suggestion
**Example request:**
```

Reply:
I have been consistent with the rest of the API docs here, we can revisit when we work on the new and improved.